### DISCUSSION TITLE
Foundry Virtual Tabletop - Version 9 Stable Release (Build 9.238)

### DISCUSSION THREAD

**Discussion by Stratlin** [cat-box/aws-foundry-ssl#36] [Q&A]
The official Foundry version 9 release notes are recommending a full reinstallation rather than an upgrade, with the stated reason being: 

>To benefit from the new Electron version and to ensure you have a clean installation[...]

Do you know if this reasoning applies to nodejs instances of Foundry like those using the steps in this wiki? If so, do you plan on releasing a patch and are/or are there additional steps that should/need to be taken? 

Thank you very much for your assistance.

**Comment by cat-box** [comment]
I believe it specifically is for deployments using electron, which this one does not. That being said, if the team at Foundry is telling us to use a clean install, I would recommend doing so. The patch [here](https://github.com/cat-box/aws-foundry-ssl/wiki/Patches#foundryvtt-update) will perform a full reinstall. Although it was originally meant for the 0.8x update, it will functionally do the same thing if you use a 0.9x download link instead.
